ext/mbstring/php_mbregex.c

```diff
@@ -922,20 +930,15 @@ static void _php_mb_regex_ereg_exec(INTERNAL_FUNCTION_PARAMETERS, int icase)
 		string_len,
 		php_mb_regex_get_mbctype_encoding()
 	)) {
-		RETURN_FALSE;
+		zend_argument_value_error(2, "must be a valid string in '%s'", php_mb_regex_get_mbctype());
```

Comment:
```suggestion
		zend_argument_value_error(2, "must be a valid string in \"%s\"", php_mb_regex_get_mbctype());
```
Would putting `encoding` to the end make sense?